### ISSUE TITLE
error on slow downloads

### ISSUE THREAD

**Issue by nsheff** [databio/refgenconf#76]
Raised by @johanneskoester

```
human_alu/fasta:default: 0.00B [00:00, ?B/s]
Traceback (most recent call last):
  File "/home/johannes/scms/snakemake-wrappers/bio/refgenie/test/.snakemake/scripts/tmp130p0ybi.wrapper.py", line 22, in <module>
    rgc.pull_asset(genome, asset, tag, force=False)
  File "/home/johannes/scms/snakemake-wrappers/bio/refgenie/test/.snakemake/conda/c95a3a6f/lib/python3.6/site-packages/refgenconf/refgenconf.py", line 564, in pull_asset
    _download_url_progress(url_archive, filepath, bundle_name, params={"tag": determined_tag})
  File "/home/johannes/scms/snakemake-wrappers/bio/refgenie/test/.snakemake/conda/c95a3a6f/lib/python3.6/site-packages/refgenconf/refgenconf.py", line 933, in _download_url_progress
    urllib.request.urlretrieve(url, filename=output_path, reporthook=dpb.update_to)
  File "/home/johannes/scms/snakemake-wrappers/bio/refgenie/test/.snakemake/conda/c95a3a6f/lib/python3.6/urllib/request.py", line 274, in urlretrieve
    reporthook(blocknum, bs, size)
  File "/home/johannes/scms/snakemake-wrappers/bio/refgenie/test/.snakemake/conda/c95a3a6f/lib/python3.6/site-packages/refgenconf/refgenconf.py", line 901, in update_to
    self.update(b * bsize - self.n)
  File "/home/johannes/scms/snakemake-wrappers/bio/refgenie/test/.snakemake/conda/c95a3a6f/lib/python3.6/site-packages/tqdm-4.7.2-py3.6.egg/tqdm/_tqdm.py", line 689, in update
ZeroDivisionError: float division by zero
```

I guess, if the download is very slow, this can happen? Maybe you could somehow catch that and handle it gracefully.

**Comment by michalstolarczyk** [comment]
TL;DR: @johanneskoester, could you try updating `tqdm` package to a more recent version?

This might be related to an issue that has been reported a few _years_ ago: https://github.com/tqdm/tqdm/pull/264. At that point the version was `<4.9.0`. The one that Johannes uses is `4.7.2`.

I've bumped up the `tqdm` requirement in `refgenconf` on `dev` some time ago because of some other problem I found, so the (probable) incompatibility of `tqdm` is just a currently released `refgenconf` problem.
